### PR TITLE
Forward after authentication attempt configuration support (#3728)

### PR DESCRIPTION
**Pull request by shazin**
Related to issue #3728

### CODE REVIEW

**Comment by rwinch** [comment]
Thanks for the PR! Do you think you could update it with some tests please :)

**Comment by shazin** [comment]
Working on it, Will push asap.

**Comment by shazin** [comment]
Tests and Minor change pushed.

**Comment by rwinch** [comment]
Thanks for the PR! This is now merged via e33e21fe6be9fac0871242b5fda2235f030afe1b

File: config/src/main/java/org/springframework/security/config/annotation/web/configurers/FormLoginConfigurer.java [hunk 1]
```diff
@@ -206,6 +209,28 @@ public FormLoginConfigurer() {
 		return this;
 	}
 
+	/**
+	 * Forward Authentication Failure Handler
+	 *
+	 * @param forwardUrl the target URL in case of failure
+	 * @return he {@link FormLoginConfigurer} for additional customization
```
Comment (php-coder) [inline config/src/main/java/org/springframework/security/config/annotation/web/configurers/FormLoginConfigurer.java]: s/he/the/ ?

File: config/src/main/java/org/springframework/security/config/annotation/web/configurers/FormLoginConfigurer.java [hunk 2]
```diff
@@ -206,6 +209,28 @@ public FormLoginConfigurer() {
...
+	public FormLoginConfigurer<H> failureForwardUrl(String forwardUrl) {
+		failureHandler(new ForwardAuthenticationFailureHandler(forwardUrl));
+		return this;
+	}
+
+	/**
+	 * Forward Authentication Success Handler
+	 *
+	 * @param forwardUrl the target URL in case of success
+	 * @return he {@link FormLoginConfigurer} for additional customization
```
Comment (php-coder) [inline config/src/main/java/org/springframework/security/config/annotation/web/configurers/FormLoginConfigurer.java]: ... and the same here.

**Pull request closed without merging**